vnext/PropertySheets/React.Cpp.props

```diff
@@ -21,7 +21,7 @@
 
   <PropertyGroup Label="Desktop">
     <!-- See https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt -->
-    <WinVer>_WIN32_WINNT_WIN8</WinVer>
```

Comment:
This could be an issue for Office, which requires Windows 8 as its minimum version.

Reply:
I take that back. The minimum version is Windows 8.1, which is actually `_WIN32_WINNT_WINBLUE `.

https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-170